### ISSUE TITLE
npm install still fails: Invalid tag name ">=^1.0.0"

### ISSUE THREAD

**Issue by tv42** [keystonejs-contrib/k6-contrib#28]
#22 & #23 were closed but this bug still very much exists.

```
$ npm i @k6-contrib/list-plugins
npm ERR! code EINVALIDTAGNAME
npm ERR! Invalid tag name ">=^1.0.0": Tags may not have any characters that encodeURIComponent encodes.
```

Workaround: `npm i --legacy-peer-deps @k6-contrib/list-plugins`

**Comment by skeary** [comment]
I think this is resolved by my PR https://github.com/keystonejs-contrib/k6-contrib/pull/27

**Comment by gautamsi** [comment]
this is closed in #29
